content/tools/data.js

```diff
@@ -101,6 +101,27 @@ const tools = [
     icons: [
       'img/icons/rainfall.svg',
     ],
+    datasets: [
+      {
+        slug: 'cb0c5f49-59d2-4f8e-8d9c-a047833f6081',
+        logo: 'scripps_100x100.png',
+      },
+      {
+        slug: '0887cba6-695c-403a-a757-674075799ea5',
+        logo: 'u_colorado.png',
+      },
+      {
+        slug: 'b8bee9f9-31c1-40d9-89a3-2dad814ba7da',
+        logo: 'gif_80x80.png'
+      }
+    ],
+    related: [
+      'extreme-weather','extreme-precipitation',
```

Comment:
Should "extreme-precipitation" be here? Isn't that the same name as this tool?

Reply:
Removed it in https://github.com/berkeley-gif/caladapt-website-2021/pull/163/commits/c1124077a07acd5fd1673d143cca56a4510f3bdd